.travis.yml

```diff
@@ -37,6 +37,8 @@ install:
   - pip install sphinx numpydoc
   # Install PEP 8 linters (flake8 is like pep8 but also reports unused imports)
   - pip install flake8
+  # Install matplotlib for benchmarks
+  - pip install matplotlib
```

Comment:
I am leaving the plot in travis. thank you for the advice, will use apt-get from now.